posts/2024/2024_07_22_kaustav_week_8.rst

```diff
@@ -0,0 +1,86 @@
+
+3. GSoC wiki posts migration (https://github.com/dipy/dipy.org/issues/21)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another important task I tackled this week was migrating our GSoC (Google Summer of Code) and GSOD (Google Season of Docs) announcements from the DIPY wiki to our main blog. This migration is part of our ongoing efforts to centralize our content and make important announcements more accessible to our community.
+
+The process involved carefully converting the wiki content into blog post format, ensuring that all the information was accurately transferred. I paid special attention to maintaining the original context and importance of these announcements while adapting them to our blog's style and format.
+
+After creating these new blog posts, I conducted thorough testing to verify that they render correctly on our development server. This included checking the formatting, ensuring all links are functional, and confirming that any associated images or embedded content displays properly.
+
+This migration will make our GSoC and GSOD announcements more visible and easily accessible to potential participants and the wider DIPY community, potentially increasing engagement with these programs.
+
+Enhancement
+-----------
+
+1. Adding sidebar to blog page (https://github.com/dipy/dipy.org/pull/53)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to addressing existing issues, I also worked on an enhancement suggested by my mentor: adding a sidebar to our `blog.html` page. Previously, our blog page lacked a sidebar, which made navigation less intuitive and user-friendly.
+
+The new sidebar is a significant improvement to our blog's usability. It provides quick access to important navigational elements such as categories, archives, and tags. This addition allows users to easily browse through our blog content, find related posts, or explore posts from specific time periods.
+
+This enhancement greatly improves the overall user experience of our blog, making it easier for readers to discover and engage with our content. It's a small change that has a big impact on the usability of our site.
+
+Difficulties & Challenges
+-------------------------
+
+I didn't particularly find anything difficult, the tasks were time consuming but not trouble causing.
+
+Next Week
+---------
+
+Looking ahead to next week, I have several objectives in mind:
+
+1. I plan to continue addressing open issues on the `dipy.org` & `dipy` repository. There are still several tasks that need attention, and I'm eager to keep up the momentum we've built. I have yet to look into which issues I will be tackling next week.
+
+2. For the preparation of next task, I have to look into Docker. I am planning to learn it through next week to be well prepared for the next task which is about github actions.
```

Comment:
Regarding this, the University of Helsinki has an [incredible (free) course on Docker](https://devopswithdocker.com/). Maybe it helps you learn the tool. It was all I needed to get started with Docker.

Reply:
Thanks for this. I will definitely require it.